apps/service-portal/src/components/Sidebar/Sidebar.css.ts

```diff
@@ -43,7 +42,7 @@ export const subnavCollapsed = style({
 export const navIcon = style({
   position: 'absolute',
   right: -theme.spacing[4],
-  top: theme.spacing[10],
+  top: 43,
```

Comment:
Shouldn't this be based on the theme? `43` seems a bit random number, it is not fitting into the multiplier of 8

Reply:
Yes it should, but than it isn't in the center 😞